target/bin/setquota

```diff
@@ -15,39 +15,33 @@ USER="${1}"
 shift
 QUOTA="${*}"
 
-function usage  { echo "Usage: setquota <user@domain> [<quota>]" ; }
+function __usage { echo 'Usage: setquota <user@domain> [<quota>]' ; }
 
-[[ -z ${USER} ]] && { usage ; _exit_with_error "no username specified" ; }
-[[ ${USER} =~ .*\@.* ]] || { usage ; _exit_with_error "username must include the domain" ; }
+[[ -z ${USER} ]] && { __usage ; _exit_with_error "No username specified" ; }
```

Comment:
Is it necessary to print `__usage` on every error? Should be removed IMO.

Reply:
I think it is because we have no `--help` flag, so every wrong usage should print how you use it correctly. Otherwise, how do you justify where to put the `__usage` and where not?

Reply:
FWIW, using python was pretty easy for little CLI tools with help generated automatically from your args. I know that you all prefer bash though, so nevermind  :stuck_out_tongue: 

https://github.com/docker-mailserver/docker-mailserver/blob/f22d75fa623675f5fa7dbb05316221dd1f02b07d/target/bin/acme_extract.py#L4-L13

Reply:
> FWIW, using python was pretty easy for little CLI tools with help generated automatically from your args. I know that you all prefer bash though, so nevermind 😛 

Nice, but I can do one better: [`clap-rs/clap`](https://github.com/clap-rs/clap) 😆 😉

Reply:
Sure if other maintainers are happy to introduce rust, some may prefer scripts over compiled binaries though.

Reply:
You are absolutely right! I initially looked at

```bash
if ! grep -qE "^${USER}\|" "${USER_DATABASE}"
then
  __usage
  _exit_with_error "User '${USER}' does not exist"
fi
```
where `__usage` is not necessary, because the correct syntax was used?

I didn't look at the error message in the other case and just concluded, that `__usage` should removed there too 😞

Reply:
Done in 30ccbb7109ec582e66d00c563f51b0a19f0974cc